<!--edit by lichunjiing 2020_12_23 -->
<launch>
  <param name="/use_sim_time" value="false" />
  <arg name="rviz" default="true" />

  <arg name="map_file_path" default="/home/alorli/test_data/nantingchechang/2022-03-21-21-24-44/map/gnss_optimization_map/" />
  <arg name="use_voxel_filter" default="false" />
  <arg name="voxel_leaf_size" default="0.07" />



  <node name="map_visualization_node"  pkg="lidar_mapping"  type="map_visualization_node" output="screen">
    <param name="map_file_path" value="$(arg map_file_path)" />
    <param name="use_voxel_filter" value="$(arg use_voxel_filter)" />
    <param name="voxel_leaf_size" value="$(arg voxel_leaf_size)" />
  </node>

  <group if="$(arg rviz)">
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find lidar_mapping)/rviz/map_visualization.rviz" />
  </group>

</launch>
